### ISSUE TITLE
Clarify language slightly

### ISSUE THREAD

**Issue by gbinal** [benbalter/is-pennsylvania-avenue-closed#7]
`NO`  - `NO, [in same large font but smaller right below] it's open.'

and 

`YES`  - `Yes, [in same large font but smaller right below] it's closed. You'll have to go around.'

**Comment by benbalter** [comment]
@gbinal any interest in taking a first pass at a pull request, even if only for the content? If you look at https://github.com/benbalter/is-pennsylvania-avenue-closed/blob/master/lib/views/index.erb#L1, I might change that to something like:

```erb
<% if closed %>
HTML REPRESENTING YES
<% else %>
HTML REPRESENTING NO
<% end %>
```

You're now a collaborator on the repo, and can even do it via the web editor, on a new branch here, if you'd like. :smile:

**Comment by gbinal** [comment]
Oy - very sorry.  I meant to do this as a PR but clicked `Commit` prematurely.  I imagine that this is why we can't have nice things.  

@benbalter - see anything wrong with this, though?  https://github.com/benbalter/is-pennsylvania-avenue-closed/commit/86a0b5074d542724fd456c72bfe6695c5c8a29de

**Comment by gbinal** [comment]
Cough.  And this - https://github.com/benbalter/is-pennsylvania-avenue-closed/commit/8ce1308b1ea858e8b30c2cadc6e22f4f03075586

**Comment by benbalter** [comment]
:+1: Now would be a good time to mention that auto deploy is enabled :smile:
